include/diaspora.php

```diff
@@ -946,11 +946,11 @@ private static function post_allow($importer, $contact, $is_comment = false) {
 			// Yes, then it is fine.
 			return true;
 		// Is it a post to a community?
-		} elseif (($contact["rel"] == CONTACT_IS_FOLLOWER) && ($importer["page-flags"] == PAGE_COMMUNITY)) {
+		} elseif (($contact["rel"] == CONTACT_IS_FOLLOWER) && ($importer["page-flags"] == PAGE_COMMUNITY) || $is_comment) {
```

Comment:
I don't like combinations of different logical operators without braces. They tend to confuse. Why not use the old code here?